velox/exec/OrderBy.cpp

```diff
@@ -92,11 +92,7 @@ OrderBy::OrderBy(
 #endif
 
   outputBatchSize_ = std::max<uint32_t>(
```

Comment:
Why can't we just use outputBatchRows()?

Reply:
I'm changing to only use outputBatchRows().